src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -610,4 +610,7 @@
   <data name="The_first_word_0_must_begin_with_a_lower_case_character" xml:space="preserve">
     <value>The first word, '{0}', must begin with a lower case character</value>
   </data>
+  <data name="File_size_exceeds_maximum_allowed_size_0" xml:space="preserve">
+    <value>File size exceeds maximum allowed size: {0}</value>
```

Comment:
Should this say what the file size was? Something like "File '{0}' size of {1} exceeds maximum allowed size of {2}"? Just so that way if a customer does hit that they have some hint what they should try getting their file under, vs. just going "uh, I need to make it smaller I guess?"